eng/common/pipelines/templates/jobs/prepare-pipelines.yml

```diff
@@ -81,16 +90,19 @@ jobs:
             $internalVariableGroups = '$(AzureSDK_Maven_Release_Pipeline_Secrets) $(Release_Secrets_for_GitHub) $(APIReview_AutoCreate_Configurations)'
           }
           "js" {
+            $generatePublicCIPipeline = 'false'
```

Comment:
@scbedd @benbp I've added a mode to disable the public jobs from being created after switching to the new pull request model.